conjure-java-core/src/main/java/com/palantir/conjure/java/types/ErrorGenerator.java

```diff
@@ -172,12 +171,35 @@ private static JavaFile generateErrorTypesForNamespace(
                 })
                 .collect(Collectors.toList());
 
+        List<MethodSpec> isRemoteExceptionDefinitions = errorTypeDefinitions.stream()
+                .map(entry -> {
+                    String typeName = CaseFormat.UPPER_CAMEL.to(
+                            CaseFormat.UPPER_UNDERSCORE, entry.getErrorName().getName());
+                    String methodName = "is" + entry.getErrorName().getName();
+
+                    String remoteExceptionVar = "remoteException";
+
+                    return MethodSpec.methodBuilder(methodName)
+                            .addModifiers(Modifier.PUBLIC, Modifier.STATIC)
+                            .addParameter(RemoteException.class, remoteExceptionVar)
+                            .returns(TypeName.BOOLEAN)
+                            .addStatement(
+                                    "return $L.name().equals($L.getError().errorName())", typeName, remoteExceptionVar)
+                            .addJavadoc(
+                                    "Returns true if the {@link RemoteException} is named $L:$L",
```

Comment:
nit: Formatting may be better using something along these lines:
```suggestion
                                    "Returns true if the {@link $T} is named {@code $L:$L}", RemoteException.class,
```